clang/test/Driver/cl-outputs.c

```diff
@@ -298,15 +298,15 @@
 // FioRACE2: "-E"
 // FioRACE2: "-o" "foo.x"
 
-// RUN: %clang_cl /Z7 /Foa.obj -### -- %s 2>&1 | FileCheck -check-prefix=ABSOLUTE_OBJPATH %s
+// RUN: %clang_cl -target x86_64-windows /Z7 /Foa.obj -### -- %s 2>&1 | FileCheck -check-prefix=ABSOLUTE_OBJPATH %s
```

Comment:
The clang-cl driver generally forces a Windows OS onto the target, but I'm not confident enough to say this isn't necessary.

Reply:
Yes, the key is `x86_64` here, but I feel safe to add the `windows` together :)